virttest/qemu_storage.py

```diff
@@ -1516,12 +1527,24 @@ def amend(self, params, cache_mode=None, ignore_status=False):
                    if key.startswith('amend_')]
         if cache_mode:
             cmd_list.append("-t %s" % cache_mode)
+        secret_objects = self._secret_objects
+        if secret_objects:
+            # add a secret object, use for adding and erasing password
+            sec_id = params["amnd_secret_id"]
+            sec_data = params["amnd_secret_data"]
```

Comment:
How about giving a brief introduction to these two new params?

Reply:
@zhencliu  Added a brief introduction to doc string, many thanks. Please recheck it. Thanks.